gradle.properties

```diff
@@ -1,5 +1,4 @@
 VERSION_CODE=29
```

Comment:
So it's actually not required, let's try to release with same version code and if it'll work we'll just drop it later